.rubocop.yml

```diff
@@ -6,7 +6,6 @@ require:
   - rubocop-rspec
 
 AllCops:
-  TargetRubyVersion: 3.2
```

Comment:
https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version

> If a `TargetRubyVersion` is not specified in your config, then RuboCop will check your project for a series of other files where the Ruby version may be specified already. The files that will be checked are (in this order): `*.gemspec`, `.ruby-version`, `.tool-versions`, and `Gemfile.lock`.